docs/test-plans/002-DAS-Benchmarks/test-cases/tc-001-x-light-finish-das-before-block-time.md

```diff
@@ -0,0 +1,25 @@
+# Test-Case #001 - Light Node Must Finish DASing in less than block time
+
+## Pre-Requisites:
+
+1. The Full Node has the latest head
+2. All light nodes are network-bootstrapped and connected to the full node (no discovery required)
+3. Share size is 32
```

Comment:
It will be awesome to have 32/64/128 matrix as we are having an assumption that it works already here #96 😅